### ISSUE TITLE
[UI] Changes to add new metadata

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/dcp-ingest-central#37]
a. Update submissions view to add a button in each tab to add a new entity to the submission
b. Redirect to an editable form (reuse the form from 1.b) and allow user to update and save the changes

**Comment by ofanobilbao** [comment]
@aaclan-ebi can we say this was already done? It is in the product backlog but the other issue that was blocked by this was already completed.

**Comment by aaclan-ebi** [comment]
Which ticket was blocked by this ticket? 

I believe this is not done yet. From what I understand, this ticket is for supporting the ability to add metadata to a project/submission in the UI. Atm, adding metadata can only be done via spreadsheets or API.

**Comment by ofanobilbao** [comment]
Thanks @aaclan-ebi!

**Comment by aaclan-ebi** [comment]
I believe this is a duplicate of ebi-ait/dcp-ingest-central#689
